README.md

```diff
@@ -402,6 +410,27 @@ createLwsIotCredentialProvider(
 freeIotCredentialProvider(&pSampleConfiguration->pCredentialProvider);
 ```
 
+### Running samples with IoT
+
+Build the samples using IoT credentials mode:
+
+```shell
+cmake .. -DIOT_CORE_ENABLE_CREDENTIALS=ON
+make
+```
+
+Set the environment variables for IoT:
+
+```shell
+export AWS_IOT_CORE_CREDENTIAL_ENDPOINT=xxxxx.credentials.iot.xxxxx.amazonaws.com
+export AWS_IOT_CORE_PRIVATE_KEY=xxxxxxxx-private.pem.key 
+export AWS_IOT_CORE_ROLE_ALIAS=xxxxxx
+export AWS_IOT_CORE_THING_NAME=xxxxxx
+export AWS_IOT_CORE_CERT=xxxxx-certificate.pem.crt
+```
+
+AWS access keys are ignored if the sample was built in IoT credentials mode.
```

Comment:
AK/SK/ST set in env?

Reply:
Correct.

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/8248241cec28b651839d91f77a706f4fd028dbeb/samples/Common.c#L759-L769